js/background.js

```diff
@@ -1451,30 +1451,32 @@ function getDebugInfo() {
     the_debug_info.settings.push("malware-notification: " + storage_get("malware-notification") + "\n");
     the_debug_info.settings = the_debug_info.settings.join("");
 
-    // Find out AdBlock version
+    // Find out CatBlock version
     var AdBlockVersion = chrome.runtime.getManifest().version;
 
-    // Is this installed build of AdBlock the official one?
+    // Is this installed build of CatBlock the official one?
+    // TODO: Replace ID's by getting info, how the extension was installed
     function AdBlockBuild() {
-        if (!SAFARI) {
-            if (chrome.runtime.id === "pljaalgmajnlogcgiohkhdmgpomjcihk") {
-                return "Beta";
-            } else if (chrome.runtime.id === "gighmmpiobklfepjocnamgkkbiglidom" ||
-                       chrome.runtime.id === "aobdicepooefnbaeokijohmhjlleamfj") {
+        if (!SAFARI && !FIREFOX) {
```

Comment:
In case of future browsers, could we not use a new `CHROME` constant instead, as this would otherwise become a `!SAFARI && !FIREFOX && !EDGE` just to check for `CHROME || OPERA`